src/Tribe/REST/V1/Attendee_Repository.php

```diff
@@ -39,7 +39,30 @@ public function __construct() {
 	 * @return WP_Query
 	 */
 	public function build_query( $use_query_builder = true ) {
-		if ( ! current_user_can( 'read_private_posts' ) ) {
+		$can_view_hidden_attendees = false;
+
+		if ( current_user_can( 'edit_users' ) || current_user_can( 'tribe_manage_attendees' ) ) {
+			$can_view_hidden_attendees = true;
+		}
+
+		/**
+		 * Whether the current user can view hidden attendees,
+		 * for instance, those that opted out of having their data public.
+		 *
+		 * @param bool         $can_view_hidden_attendees The default value for whether the user can view hidden attendees.
+		 * @param null|WP_User $user                      The WP_User instance, if available.
+		 *
+		 * @since TBD
+		 *
+		 * @return bool Whether the current user can view hidden attendees.
+		 */
+		$can_view_hidden_attendees = (bool) apply_filters(
+			"tribe_tickets_user_can_view_hidden_attendees_rest",
```

Comment:
:no_entry_sign: String "tribe_tickets_user_can_view_hidden_attendees_rest" does not require double quotes; use single quotes instead

<sub>hash: 160447488450fb6f34fea1fb24602520</sub>